src/behaviors/Behavior.js

```diff
@@ -6,7 +6,7 @@ var Base = require('../lib/Base');
 
 var Column = require('./Column');
 var dialogs = require('../dialogs');
-var CellProvider = require('../lib/CellProvider');
+var CellProvider = require('../cellRenderers/CellProvider');
```

Comment:
That the location of cellrenders would be a little clearer. CellProvider in lib doesn't strike as the first place we are calling cell rendering. This is also consistent with celleditor structure. Even more so many of files in lib should be broken out into utils and lib but I didn't want to go overboard